query/stdlib/influxdata/influxdb/source.go

```diff
@@ -45,7 +45,9 @@ type Source struct {
 func (s *Source) Run(ctx context.Context) {
 	labelValues := s.m.getLabelValues(ctx, s.orgID, s.op)
 	start := time.Now()
-	err := s.runner.run(ctx)
```

Comment:
Is this supposed to be optionally enabled based on the experimental tracing?

Reply:
Waiting for the new flux release to have the isExperimentalTracingEnabled() API exposed.